2021-04-18 21:59:23,751 INFO    Thread-3  :16969 [wandb_setup.py:_flush():69] setting env: {'project': 'simpledrawer-ddqn_random_start_fixed_line_pos', 'entity': 'riccir', 'root_dir': '/home/antoniopioricciardi/PycharmProjects/simpledrawer', 'sweep_id': '85yggpre', 'run_id': 'jf7lr6ai', 'sweep_param_path': '/home/antoniopioricciardi/PycharmProjects/simpledrawer/wandb/sweep-85yggpre/config-jf7lr6ai.yaml'}
2021-04-18 21:59:23,751 INFO    Thread-3  :16969 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-18 21:59:23,751 INFO    Thread-3  :16969 [wandb_init.py:_log_setup():336] Logging user logs to /home/antoniopioricciardi/PycharmProjects/simpledrawer/wandb/run-20210418_215923-jf7lr6ai/logs/debug.log
2021-04-18 21:59:23,751 INFO    Thread-3  :16969 [wandb_init.py:_log_setup():337] Logging internal logs to /home/antoniopioricciardi/PycharmProjects/simpledrawer/wandb/run-20210418_215923-jf7lr6ai/logs/debug-internal.log
2021-04-18 21:59:23,752 INFO    Thread-3  :16969 [wandb_init.py:init():369] calling init triggers
2021-04-18 21:59:23,752 INFO    Thread-3  :16969 [wandb_init.py:init():374] wandb.init called with sweep_config: {'batch_size': 32, 'fc_layer_size': 128, 'gamma': 0.6, 'learning_rate': 0.001}
config: {'replace': 1000, 'learning_rate': 0.001, 'gamma': 0.6, 'epsilon': 0.7, 'epsilon_min': 0.0, 'epsilon_dec': 1e-05, 'mem_size': 50000, 'batch_size': 32, 'optimizer': 'adam', 'fc_layer_size': 128, 'max_steps': 200000}
2021-04-18 21:59:23,752 INFO    Thread-3  :16969 [wandb_init.py:init():418] starting backend
2021-04-18 21:59:23,752 INFO    Thread-3  :16969 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-18 21:59:23,758 INFO    Thread-3  :16969 [backend.py:ensure_launched():123] starting backend process...
2021-04-18 21:59:23,761 INFO    Thread-3  :16969 [backend.py:ensure_launched():127] started backend process with pid: 17004
2021-04-18 21:59:23,761 INFO    Thread-3  :16969 [wandb_init.py:init():423] backend started and connected
2021-04-18 21:59:23,762 INFO    Thread-3  :16969 [wandb_run.py:_config_callback():749] config_cb None None {'batch_size': 32, 'fc_layer_size': 128, 'gamma': 0.6, 'learning_rate': 0.001}
2021-04-18 21:59:23,763 INFO    Thread-3  :16969 [wandb_init.py:init():463] updated telemetry
2021-04-18 21:59:23,763 INFO    Thread-3  :16969 [wandb_init.py:init():482] communicating current version
2021-04-18 21:59:24,292 INFO    Thread-3  :16969 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.26 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-18 21:59:24,292 INFO    Thread-3  :16969 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-18 21:59:24,595 INFO    Thread-3  :16969 [wandb_init.py:init():520] starting run threads in backend
2021-04-18 21:59:24,620 INFO    Thread-3  :16969 [wandb_run.py:_console_start():1524] atexit reg
2021-04-18 21:59:24,620 INFO    Thread-3  :16969 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-18 21:59:24,620 INFO    Thread-3  :16969 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-18 21:59:24,621 ERROR   Thread-3  :16969 [wandb_run.py:_redirect():1441] Failed to redirect.
Traceback (most recent call last):
  File "/home/antoniopioricciardi/PycharmProjects/simpledrawer/venv/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1434, in _redirect
    out_redir.install()
  File "/home/antoniopioricciardi/PycharmProjects/simpledrawer/venv/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 605, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/antoniopioricciardi/PycharmProjects/simpledrawer/venv/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 555, in add_fd
    self._register()
  File "/home/antoniopioricciardi/PycharmProjects/simpledrawer/venv/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 540, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2021-04-18 21:59:24,621 INFO    Thread-3  :16969 [wandb_init.py:init():544] run started, returning control to user process
2021-04-18 21:59:31,326 INFO    Thread-3  :16969 [wandb_run.py:finish():1167] finishing run riccir/simpledrawer-ddqn_random_start_fixed_line_pos/jf7lr6ai
